app/Jobs/Data/Oyster/LinkOysterJourneyEventsJob.php

```
@@ -0,0 +1,171 @@
+                }
+
+                // Get the station objects to compare
+                $touchedInTarget = $touchedInEvent->target;
+                $touchedOutTarget = $touchedOutEvent->target;
+
+                if (! $touchedInTarget || ! $touchedOutTarget) {
+                    continue;
+                }
+
+                // Normalize station names for comparison
+                $originName = strtolower(trim($touchedInTarget->title ?? ''));
```

PHPMD (2.15.0)

Avoid unused local variables such as '$originName'. (undefined)

(UnusedLocalVariable)